tsl/test/sql/include/transparent_decompression_ordered_index.sql

```diff
@@ -138,3 +138,70 @@ SET enable_mergejoin = TRUE;
 SET enable_hashjoin = TRUE;
 
 --end github issue 1558
+
+-- github issue 2673 
+-- nested loop join with parameterized path
+-- join condition has a segment by column and another column.
+SET enable_hashjoin = false;
+SET enable_mergejoin=false;
+SET enable_material = false;
+SET enable_seqscan = false;
+
+-- restrict so that we select only 1 chunk.
+:PREFIX
+WITH lookup as ( SELECT * from (values( 3, 5) , (3, 4) ) as lu( did, version) )
+SELECT met.*, lookup.*
+FROM metrics_ordered_idx met join lookup
+ON met.device_id = lookup.did and met.v0 = lookup.version
+WHERE met.time > '2000-01-19 19:00:00-05' 
+      and met.time < '2000-01-20 20:00:00-05'; 
+
+--add filter to segment by (device_id) and compressed attr column (v0)
+:PREFIX
+WITH lookup as ( SELECT * from (values( 3, 5) , (3, 4) ) as lu( did, version) )
+SELECT met.*, lookup.*
+FROM metrics_ordered_idx met join lookup
+ON met.device_id = lookup.did and met.v0 = lookup.version
+WHERE met.time > '2000-01-19 19:00:00-05' 
+      and met.time < '2000-01-20 20:00:00-05' 
+      and met.device_id = 3 and met.v0 = 5;
+
+:PREFIX
+WITH lookup as ( SELECT * from (values( 3, 5) , (3, 4) ) as lu( did, version) )
+SELECT met.*, lookup.*
+FROM metrics_ordered_idx met join lookup
+ON met.device_id = lookup.did and met.v0 = lookup.version 
+WHERE met.time = '2000-01-19 19:00:00-05' 
+      and met.device_id = 3
+      and met.device_id_peer = 3 and met.v0 = 5;
+
+-- lateral subquery
+:PREFIX
+WITH f1 as ( SELECT * from (values( 7, 5, 4) , (4, 5, 5) ) as lu( device_id, device_id_peer, v0) )
+SELECT * FROM  metrics_ordered_idx met 
+JOIN LATERAL
+  ( SELECT node, f1.* from nodetime , f1
+    WHERE  node = f1.device_id) q
+ON met.device_id = q.node and met.device_id_peer = q.device_id_peer 
+   and met.v0 = q.v0 and met.v0 > 2 and time = '2018-01-19 20:00:00-05';
+
+-- filter on compressed attr (v0) with seqscan enabled and indexscan 
+-- disabled. filters on compressed attr should be above the seq scan.
+SET enable_seqscan = true;
+SET enable_indexscan = false;
+:PREFIX
+WITH lookup as ( SELECT * from (values( 3, 5) , (3, 4) ) as lu( did, version) )
+SELECT met.*, lookup.*
+FROM metrics_ordered_idx met join lookup
+ON met.device_id = lookup.did and met.v0 = lookup.version 
+   and met.device_id = 3
+WHERE met.time > '2000-01-19 19:00:00-05' 
+      and met.time < '2000-01-20 20:00:00-05' 
+      and met.device_id = 3
+      and met.device_id_peer = 3 and met.v0 = 5;
+
+SET enable_hashjoin = true ;
+SET enable_mergejoin= true;
+SET enable_material = true;
+SET enable_indexscan = true;
```

Comment:
should use RESET here to reset them back to default values

Reply:
fixed